assignments/callbacks.js

```diff
@@ -23,26 +23,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
+  return cb(arr.length);
   // getLength passes the length of the array into the callback.
 }
 
+getLength(items, function(length) {
+  console.log(length)
+});
+
 function last(arr, cb) {
+  return cb(arr[arr.length-1]);
   // last passes the last item of the array into the callback.
 }
 
+last(items, function(ultimo) {
+  console.log(ultimo)
+});
+
 function sumNums(x, y, cb) {
+  return cb(x + y);
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
 
+sumNums(7,10, function(sum){
+  console.log(sum)
+});
+
+
 function multiplyNums(x, y, cb) {
+  return cb(x*y);
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
 
+multiplyNums(7,7, function(product){
+  console.log(product)
+});
+
 function contains(item, list, cb) {
+  for (let i=0; i<arr.length; i++){
```

Comment:
```suggestion
  for (let i=0; i<item.length; i++){
```
remember if you look at the parameters you're array is `item` so you have to use that